ios/Classes/Module/FacebookModule.m

```diff
@@ -370,14 +370,24 @@ - (void)presentSendRequestDialog:(NSArray<NSDictionary<NSString *, id> *> *)args
 {
   NSDictionary *_Nonnull params = [args objectAtIndex:0];
 
+  NSDictionary *_Nonnull dataParams = nil;
+  if (args.count >= 2) {
+    dataParams = [args objectAtIndex:1];
+  }
+
   NSString *message = [params objectForKey:@"message"];
   NSString *title = [params objectForKey:@"title"];
   NSArray *to = [params objectForKey:@"to"];
   NSArray *recipients = [params objectForKey:@"recipients"];
   NSArray *recipientSuggestions = [params objectForKey:@"recipientSuggestions"];
   FBSDKGameRequestFilter filters = [TiUtils intValue:[params objectForKey:@"filters"]];
   NSString *objectID = [params objectForKey:@"objectID"];
-  NSString *data = [params objectForKey:@"data"];
+  NSString *data = nil;
+  if (dataParams) {
+    data = [dataParams objectForKey:@"data"];
+  } else {
+    data = [params objectForKey:@"data"];
+  }
```

Comment:
Same as above. This will introduce a second parameter which is unnecessary. Simply update all the example so they correctly use the existing `data` property.

For example:

```js
fb.presentSendRequestDialog({
  message: 'Go to https://appcelerator.com/',
  data: { badge_of_awesomeness: '1', social_karma: '5' }
});
```

Reply:
Thanks, its addressed in latest commit. Please have a look.